fetchGrammarKit.sh

```diff
@@ -8,8 +8,8 @@ GrammarKitVersion="1.2.0"
         cp ~/Tools/GrammarKit-${GrammarKitVersion}.zip . &&
         cp ~/Tools/light-psi-all-${GrammarKitVersion}.jar .
     else
-        `wget -O GrammarKit-${GrammarKitVersion}.zip https://github.com/JetBrains/Grammar-Kit/releases/download/${GrammarKitVersion}/GrammarKit.zip`&&
-        `wget -O light-psi-all-${GrammarKitVersion}.jar https://github.com/JetBrains/Grammar-Kit/releases/download/${GrammarKitVersion}/light-psi-all.jar` &&
+        `wget --quiet -O GrammarKit-${GrammarKitVersion}.zip https://github.com/JetBrains/Grammar-Kit/releases/download/${GrammarKitVersion}/GrammarKit.zip`&&
```

Comment:
I'm not so sure about the "--quiet".  Do we still get appropriate error info -- and does the build stop -- when wget fails?